app/views/layouts/application.html.erb

```diff
@@ -5,11 +5,35 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
   </head>
 
-  <body>
-    <%= yield %>
+  <body class="h-screen">
+    <header class="bg-gray-800 text-orange-50">
+        <nav class="h-16 flex justify-between items-center w-[96%] mx-auto">
+          <div>Stochastic Soccer</div>
+          <div class="">
+            <ul class="flex items-center gap-8">
+              <li class="">
+                <%= link_to "Standings", root_path, class: "hover:text-orange-300" %>
+              </li>
+              <li class="">
+                <%= link_to "Matches", matches_path, class: "hover:text-orange-300" %>
+              </li>
+              <li class="">
+                <%= link_to "Simulations", simulations_path, class: "hover:text-orange-300" %>
+              </li>
+            </ul>
+          </div>
+          <div class="">
+            <buttom class="bg-blue-800 text-white px-5 py-2 rounded-full hover:bg-blue-400">Sign in</buttom>
+          </div>
+        </nav>
+    </header>
```

Comment:
You can put this in a partial, as well, to encapsulate the styles for the header.

https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials

Reply:
Nice. I love improved readibility.

Done at #21